classes/helpers/FrmEmailSummaryHelper.php

```diff
@@ -0,0 +1,529 @@
+	 */
+	public static function set_last_sent_date( $type, $value = null ) {
+		$options = self::get_options();
+
+		$options[ 'last_' . $type ] = null === $value ? gmdate( 'Y-m-d' ) : '';
+		self::save_options( $options );
+	}
+
+	/**
+	 * Gets the created date of earliest form.
+	 *
+	 * @return string
+	 */
+	private static function get_earliest_form_created_date() {
+		return FrmDb::get_var(
+			'frm_forms',
+			array(),
+			'created_at',
+			array( 'order_by' => 'id ASC' )
+		);
+	}
+
+	/**
+	 * Gets payments data.
+	 *
+	 * @param string $from_date From date.
+	 * @param string $to_date   To date.
+	 * @return array            Contains `count` and `total`.
+	 */
+	public static function get_payments_data( $from_date, $to_date ) {
+		$data = array(
+			'count' => 0,
+			'total' => array(),
+		);
+
+		$payments = FrmDb::get_results(
+			'frm_payments',
+			array(
+				'created_at >' => $from_date,
+				'created_at <' => $to_date . ' 23:59:59',
+				'status'       => 'complete',
+			),
+			'action_id,amount'
+		);
+
+		if ( ! $payments ) {
+			return $data;
+		}
+
+		$data['count'] = count( $payments );
+		$data['total'] = self::get_payment_total_data( $payments );
+
+		return $data;
+	}
+
+	/**
+	 * Gets payment total data.
+	 *
+	 * @param object[] $payments Array of payment objects.
+	 * @return array Return array of total amount for each currency.
+	 */
+	private static function get_payment_total_data( $payments ) {
+		$data = array();
+		foreach ( $payments as $payment ) {
+			list( $amount, $currency ) = FrmTransLiteAppHelper::get_amount_and_currency_from_payment( $payment );
+
+			if ( ! isset( $data[ $currency ] ) ) {
+				$data[ $currency ] = 0;
+			}
+
+			$data[ $currency ] += floatval( $amount );
+		}
+
+		return $data;
+	}
+
+	/**
+	 * Gets entries count in a date range.
+	 *
+	 * @param string $from_date From date.
+	 * @param string $to_date   To date.
+	 * @return int
+	 */
+	public static function get_entries_count( $from_date, $to_date ) {
+		return FrmDb::get_count(
+			'frm_items',
+			array(
+				'created_at >' => $from_date, // The `=` is added after `>` in the query.
+				'created_at <' => $to_date . ' 23:59:59',
+				'is_draft'     => 0,
+			)
+		);
+	}
+
+	/**
+	 * Gets top forms in a date range.
+	 *
+	 * @param string $from_date From date.
+	 * @param string $to_date   To date.
+	 * @param int    $limit     Limit the result. Default is 5.
+	 * @return array            Contains `form_id`, `form_name`, and `items_count`.
+	 */
+	public static function get_top_forms( $from_date, $to_date, $limit = 5 ) {
+		global $wpdb;
+
+		return $wpdb->get_results(
+			$wpdb->prepare(
+				"SELECT fr.id AS form_id, fr.name AS form_name, COUNT(*) as items_count
+						FROM {$wpdb->prefix}frm_items AS it INNER JOIN {$wpdb->prefix}frm_forms AS fr ON it.form_id = fr.id
+						WHERE it.created_at BETWEEN %s AND %s AND it.is_draft = 0
+						GROUP BY form_id ORDER BY items_count DESC LIMIT %d",
+				$from_date,
+				$to_date . ' 23:59:59',
```

Comment:
I'm curious about timezones here as well.

This is always UTC, right?

Could we localize it?

Reply:
I'm not sure if we need the local timezone or not. Maybe a bit of statistical data goes to the next or previous month, which I think it's acceptable. @stephywells What do you think about this?

Reply:
I don't think it's worth a lot of time, but I think it would be quick so we might as well. I think this should do the job:
```suggestion
				FrmAppHelper::get_localized_date( 'Y-m-d', $to_date . ' 23:59:59' ) ,
```

Reply:
No.... that's not what you're saying. Yes! That would make more sense.